## Welcome

My name is Kiley Hewitt. I am a Junior double majoring in Computer Science and Biology at Allegheny College. I am very interested in the interdisciplinary applications of Computer Science in research and in product design. As a board member of Women in Science and Association for Computing Machinery, and as a fairly regular participant in academic department and other community events, I am also passionate about Science outreach and building community around Science. I am looking to build my experience in interdisciplinary areas of Science through research and internships throughout my undergraduate education before attending graduate school for a subject related to Computational Biology.

## Projects

[Chronic Disease Risk Calculator](https://github.com/Allegheny-Computer-Science-202-S2020/cs202s2020-final-project-hewittk)

[Methods of Developing Causal Gene Regulatory Networks of Dynamic Single Cell Data](<./Kiley Hewitt TECBio Poster.pdf>)

[Metabolic Proteins in Climate Regulation](https://github.com/allegheny-bioinformatics-300-f2019/project-warbler/blob/master/writing/finalReport/finalReport.md)

## Research

Over the summer of 2020, I participated in the TECBio REU at University of Pittsburgh. I worked in the Benos lab on challenging different causal inference assumptions to feasibly map gene regulatory networks. The first assumption that I worked to challenge was the assumption of stationarity, that variables in a graph are in equilibrium and do not change over time.

Over the summer of 2020, I conducted Computational Biology research through the TECBio REU at the University of Pittsburgh. My project was to challenge existing methods of learning causal graphical models to better suit gene regulatory networks to be learned from the dynamic single cell data that gene expression tracking methods often provide. I did this by using various methods such as RNA velocity analysis, Graphical Lasso regression, and monitoring of accuracy through different correlation metrics to make undirected graphs with suitable connectivity to be learned by the labâ€™s causal inference algorithms.

The first causal inference assumption that I challenged was the assumption of stationarity stating that variables in a graph are in equilibrium and do not change over time which makes it traditionally unsuitable for learning gene regulatory networks of datasets with dynamic gene expression. I challenged this assumption by creating undirected graphs in which the latent time is seen as a dynamic variable and the expression of some genes in a pathway are mapped as a descendent of time. The second causal inference assumption that I challenged was the assumption of linearity stating that variables in a graph have only linear, monotonic relationships which is often untrue of the signaling effects between genes. I challenged this assumption by using Nonparanormal Data Transform to make any consistent increase or decrease shown in gene expression comprehensible by the linear regression tools that build the undirected graph.

## Contact
[Linkedln](https://www.linkedin.com/in/kiley-hewitt-3babaa197/)

[Github](https://github.com/hewittk)

Email: hewittk@allegheny.edu

